apps/crn-server/test/fixtures/news.fixtures.ts

```diff
@@ -39,19 +80,26 @@ export const newsSquidexApiResponse: {
   ],
 };
 
-export const getNewsDataObject = (): NewsDataObject => ({
+export const getNewsDataObject = (
+  isContentfulResponse = false,
+): NewsDataObject => ({
   id: 'news-1',
   title: 'News 1',
   type: 'News',
+  frequency: 'News Articles',
   shortText: 'Short text of news 1',
   text: '<p>text</p>',
-  thumbnail: `${baseUrl}/api/assets/${appName}/thumbnail-uuid1`,
+  thumbnail: isContentfulResponse
+    ? `http://images.ctfassets.net/thumbnail-id.png`
+    : `${baseUrl}/api/assets/${appName}/thumbnail-uuid1`,
   created: '2020-09-08T16:35:28.000Z',
 });
 
-export const getListNewsDataObject = (): ListNewsDataObject => ({
+export const getListNewsDataObject = (
+  isContentfulResponse = false,
```

Comment:
again not a huge fan of this. Having to choose which data source we are getting this data object from when we said data objects should be the same for both sources is a little bit of an anti pattern to me